src/main/java/org/minions/devfund/noemiguzman/moviedatabase/MovieDatabase.java

```diff
@@ -0,0 +1,142 @@
+package org.minions.devfund.noemiguzman.moviedatabase;
+
+import java.util.ArrayList;
+/**
+ * class movie data base.
+ */
+public class MovieDatabase {
+    private ArrayList<Movie> movieList;
+    private ArrayList<Actor> actorList;
+
+    /**
+     * Initializes the MovieDatabase object.
+     */
+    public MovieDatabase() {
+        this.movieList = new ArrayList<>();
+        this.actorList = new ArrayList<>();
+    }
+
+   /**
+     * Getter for the movies in the database.
+     * @return  all the movies in the bd
+     */
+    public ArrayList<Movie> getMovieList() {
+        return this.movieList;
+    }
+
+    /**
+     * Getter for the actors in the database.
+     * @return an ArrayList of all the actors in the
+     * database
+     */
+    public ArrayList<Actor> getActorList() {
+        return this.actorList;
+    }
+
+    /**
+     * Getter for the String names of the movies.
+     * @return ArrayList of Strings of the names of all
+     * the movies in the database.
+     */
+    private ArrayList<String> getMovieNames() {
+        ArrayList<String> movieNames = new ArrayList<>();
+        for (Movie movie: this.movieList) {
+            movieNames.add(movie.getName());
+        }
+        return movieNames;
+    }
+
+
+    /**
+     * Add a movie to the database.
+     * @param name String for the movie's name
+     * @param actors Array of Strings for the actors'
+     *               names.
+     */
+    public void addMovie(final String name, final String[] actors) {
+        if (!this.getMovieNames().contains(name)) {
+            ArrayList<Movie> movieList = new ArrayList<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "movieList" which hides the field declared at line 8. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)